browser/ui/webui/brave_federated/federated_internals.mojom

```diff
@@ -5,12 +5,15 @@
 
 module federated_internals.mojom;
 
-struct AdStoreLog {
-    int32 log_id;
-    double log_time;
-    string log_locale;
-    int32 log_number_of_tabs;
-    bool log_label;
+import "brave/components/brave_federated/public/interfaces/brave_federated.mojom";
+
+struct InternalsCovariate {
+  int32 training_instance_id;
+  brave_federated.mojom.Covariate covariate;
+};
+
+struct InternalsTrainingInstance {
```

Comment:
Could we not just use `array<...>` throughout the codebase instead of having a struct for a single member